### PR TITLE
Clarify how multiple MR/PR IDs should be ordered in config variable traceability_checklist

### PR DESCRIPTION
**Pull request by JasperCraeghs**
Partially answers https://github.com/melexis/sphinx-traceability-extension/issues/272

`traceability_checklist['merge_request_id']` shall contain one or more comma-separated IDs of a merge/pull request. The order of these IDs matters. They are used to send sequential queries to GitLab/GitHub with each new response possibly overwriting the response of a previous query.

Imagine a configuration `traceability_checklist['merge_request_id'] = '3,1,9'`. If only the descriptions of MR 3 and 1 contain a specific `checklist-item`'s ID, the checkbox value in the desription of MR 1 will be used to determine and set the attribute value of that `checklist-item`.

However, I have never encountered a use case that lead me to configure multiple merge requests that contain the same checklist-item(s). I've always only configured the most recent/relevant merge request because, for me, a new checklist makes the old one completely obsolete. Your use case may vary.

FYI @passeris

### CODE REVIEW

**Comment by Letme** [comment]
Request was also to provide an example conf.py structure. I assume with the incorporated config from python decouple import?

**Comment by passeris** [comment]
In a few ASPICE assessments for example we were asked to prove that we actually completed a checklist. For now, I can refer to an MR that shows the checkboxes checked. 

If I understand your use-case correctly, then we would have a design document that we review using the checklist. We could demonstrate that we used the checklist on our latest review but the previous ones will be overwritten in that sense. That could be okay, because as long as we demonstrate that we have gone through the entire checklist for the latest release of the document, we should be okay.

**Comment by Letme** [comment]
With this checklist it is already embedded into your documentation (pdf or HTML) - MR is indeed the proof and log who clicked it (and when). So only thing you need to make sure is that tagged documentation is always pushed to separate folder, everything else is done by our extension.

The checklist already fills the document attributes so after you do the review from the MR it is rendered into your documentation with answers. Also if you re-run the pipeline later on, the checklist will be rendered again, so make sure tags are separated from branches/latest as that is how you are able to demonstrate that you checked everything when it was required.

I am glad someone else is using this also for ISO26262 and ASPICE certification reviews.

File: doc/usage.rst [hunk 1]
```diff
@@ -723,14 +723,14 @@ GitLab
 - *PRIVATE_TOKEN* is your personal access token that has API access.
 - *API_HOST_NAME* is the host name of the API, e.g. *https://gitlab.example.com/api/v4*
 - *PROJECT_ID* is the ID of the project.
-- *MERGE_REQUEST_ID* is the internal ID of the merge request.
+- *MERGE_REQUEST_ID* are one or more internal IDs of merge requests (comma-separated) ordered from low to high priority.
```
[the remark this answers is not in the document]
Comment (JasperCraeghs) [reply doc/usage.rst]: Like this? 339ab9c9b89eb69d89ddae7d01bb6acdb4f61ffc

**Pull request merged**